Gulpfile.js

```diff
@@ -22,17 +22,28 @@ gulp.task('clean-win-bin', function (cb) {
     del('bin/win', cb);
 });
 
-gulp.task('build-win-executables', ['clean-win-bin'], function () {
+gulp.task('build-win-dll', ['clean-win-bin'], function () {
     return gulp
-        .src('src/natives/**/*.csproj')
+        .src('src/natives/**/shared.csproj')
         .pipe(msbuild({
-            targets: ['Clean', 'Build']
+            targets:     ['Clean', 'Build'],
+            errorOnFail: true
+        }));
+});
+
+gulp.task('build-win-executables', ['build-win-dll'], function () {
+    return gulp
+        .src(['!src/natives/**/shared.csproj', 'src/natives/**/*.csproj'])
```

Comment:
ok, I got it. Maybe `build-win-dlls` then? Or `build-win-util-dll`. Now it looks like `build-some-noname-win-dll` ))